drivers/wallclock/sun6i-cnt64.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#include <compiler.h>
+#include <error.h>
+#include <mmio.h>
+#include <util.h>
+#include <drivers/clock.h>
+#include <drivers/wallclock.h>
+
+#define  OSC24M_CNT64_CTRL_REG 0x280
```

Comment:
Not sure why `uncrustify` didn't catch it, but there's still an extra space here